WorkingInR/2_First_Steps_in_R.Rmd

```diff
@@ -302,28 +318,38 @@ Read more about matrices [here](https://www.rdocumentation.org/packages/base/ver
+A data frame is a two-dimensional object that stores data of **various** types in the form of a table. 
+Data frames are a popular way to store research data since the columns are usually associated with single variables (and are thus of a specific data type such as numeric or character) and rows are associated with observations.
 
-It is fairly easy to create a data frame by using the function `data.frame()` to put vectors or factors side-by-side,  vertically, and in a given sequence. However you must make sure that these vectors have the same length (i.e. the same number of elements) since a data frame cannot have empty cells. As a result of using `data.frame()`, the column names will match the name of the vectors. You can also specify a name for a column using `=`.
+It is fairly easy to create a data frame by using the function `data.frame()` to put vectors or factors side-by-side, vertically, and in a given sequence. 
+However you must make sure that these vectors have the same length (i.e. the same number of elements) since a data frame cannot have empty cells. 
+As a result of using `data.frame()`, the column names will match the name of the vectors. 
+You can also specify a name for a column using `=`.
 
 ```{r df, echo=TRUE}
 df <- data.frame(1:12, month.abb, "Month" = month.name)
 df 
 ```
 
-When printing a data frame (like in the example above) to visually evaluate it, the whole object is displayed, including row numbers/names and column names. This means that several hundred lines or observations may suddenly invade your screen as soon as you run the name of the object in R. You will find some useful functions to shortly describe a huge data frame further below in Chapter \@ref(describing-an-object).
+When printing a data frame (like in the example above) to visually evaluate it, the whole object is displayed, including row numbers/names and column names. 
+This means that several hundred lines or observations may suddenly invade your screen as soon as you run the name of the object in R.
+You will find some useful functions to shortly describe a huge data frame further below in Chapter \@ref(describing-an-object).
 
 Read more about data frames [here](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/data.frame).
 
 ### Tibbles
 
-A tibble is an evolution of the data frame described above. It is not part of the R base package (a.k.a. base R), but comes with the package `tidyverse`, which you have to install/load ahead of time.
+A tibble is an evolution of the data frame described above. 
+It is not part of the R base package (a.k.a. base R), but comes with the package `tidyverse`, which you have to install/load ahead of time.
 
 ```{r load tydiverse, echo=TRUE}
 library(tidyverse)
 ```
 
```

Comment:
L775: You may thus understand why tibbles are a much better choice than data frame, especially when handling large data sets, with hundreds of rows and columns.
Tibbles _print_ much better than data frames do!

Maybe we do not have to say it so strongly, because some people might just like dataframes better...
I would just state:
Tibbles print much better and give more information than dataframes. We do not have to add that they are the better choice.

L798 #### Visualizing data with View()
I don't think you need this section. It was already mentioned above.

L812: Creating a tibble from scratch
this one is also not needed. You created a tibble above.